src/css/components/_keyboard.scss

```diff
@@ -91,6 +100,22 @@
     overflow-x: scroll;
 }
 
+.dark-theme .keyboard .auto-complete-list-item {
+    display: flex;
+    color: #fff;
+    font-size: 24px;
+    padding: 10px;
+    background-color: black;
+    margin-right: 5px;
+    border-radius: 10px;
+    white-space: nowrap;
+    cursor: pointer;
+    user-select: none;
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+}
```

Comment:
Might suggest moving some of these common properties out of this definition to make it easier to maintain:

```
.keyboard .auto-complete-list-item {
    display: flex;
    font-size: 24px;
    padding: 10px;
    margin-right: 5px;
    border-radius: 10px;
    white-space: nowrap;
    cursor: pointer;
    user-select: none;
    -webkit-touch-callout: none;
    -webkit-user-select: none;
    -moz-user-select: none;
}

.dark-theme .keyboard .auto-complete-list-item {
    color: #fff;
    background-color: black;
}

.light-theme .keyboard .auto-complete-list-item {
    color: #000;
    background-color: #fff;
}
```